### PR TITLE
add: test unit for reverse string

### PR DESCRIPTION
**Pull request by iamabdh**
https://trello.com/c/0irt4Sgd

### CODE REVIEW

File: src/main/java/io/code/CodeExample.java [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
 
+import java.util.ArrayList;
+
 public class CodeExample {
 	
 	public static void main(String[] arg) {
-		
 	}
 	
 	public int DummyFunction(int a) {
```
Comment (AmeerAllawati) [inline src/main/java/io/code/CodeExample.java]: Maybe delete this function.

**Review by AmeerAllawati — APPROVED** [review]

**Review by CodelineAtyab — APPROVED** [review]
Conditional Approval - Need to make the suggested changes.

**Review by AnwaarAlbusaidi — APPROVED** [review]
good job

File: src/main/java/io/code/CodeExample.java (hunk 1 above)
Comment (iamabdh) [reply src/main/java/io/code/CodeExample.java]: Ok.

**Review by BaraahAlKalbani — APPROVED** [review]

**Pull request merged**